src/steamship/agents/context.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, Any, List, Optional
+
+from pydantic import BaseModel
+
+from steamship import Steamship
+from steamship.experimental.transports.chat import ChatMessage
+
+
+class ChatHistory(BaseModel):
+    chat_id: str = ""  # chat history is scoped by a chat_id. chat_id is unique across user_id's
+    messages: List[str] = []  # Represents a database that can select & sort messages
+
+    def get_history(self, message: ChatMessage, context):
+        return self.messages
+
+    def add_user_message(self, message: ChatMessage):
```

Comment:
nit, but maybe broader design principle -- I think we might want to always work in `List[Block]` instead of just `Block` just so that we avoid having to switch back and forth in different places in the code. This feels like on the spectrum of aesthetic preference though